### PR TITLE
start pathways post, add logos image w prereview to events page

### PR DESCRIPTION
**Pull request by stefaniebutland**

### CODE REVIEW

File: blog/2024-01-16-pathways-2024/index.qmd [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+---
+
+*Quicklinks:*
+
+-   *Pathways to Open Science [website](https://openscapes.github.io/pathways-to-open-science/)*
+-   [*Registration*](https://zoom.us/meeting/register/tJ0ldempqDIrHtQ4KqAbr442Tm0xl4-h_dtC) *(free)*
+
+------------------------------------------------------------------------
+
+We're thrilled and grateful to announce our 2024 offering of Pathways to Open Science! Thanks to new funding from the [Mozilla Foundation's Alumni Connection grants](https://foundation.mozilla.org/en/blog/mozilla-announces-first-ever-alumni-connection-grant-finalists/), the Openscapes team led by Ileana Fenwick, with Dr Alex Davis and Aneese Williams, is excited to collaborate with [PREreview](https://prereview.org/) and Dr Antoinette Foster to bring expertise in anti-racist scholarly review to the programming this year. Pathways to Open Science is a free remote event series for Black environmental & marine researchers to build community for the future of data intensive science.
```
Comment (jules32) [inline blog/2024-01-16-pathways-2024/index.qmd:27]:
```suggestion
We're thrilled and grateful to announce our 2024 offering of Pathways to Open Science! This is an ongoing collaboration with [Black in Marine Science (BIMS)](https://www.blackinmarinescience.org/), [Black Women in Ecology, Evolution, and Marine Science (BWEEMS)](https://www.bweems.org/), and HBCU marine science professors led by led by Ileana Fenwick, with Dr Alex Davis and Aneese Williams. Thanks to new funding from the [Mozilla Foundation's Alumni Connection grants](https://foundation.mozilla.org/en/blog/mozilla-announces-first-ever-alumni-connection-grant-finalists/), the Openscapes team is excited to collaborate with [PREreview](https://prereview.org/) and Dr Antoinette Foster to bring expertise in anti-racist scholarly review to the programming this year. Pathways to Open Science is a free remote event series for Black environmental & marine researchers to build community for the future of data intensive science.
```

**Comment by jules32** [comment]
Hi @stefaniebutland ! I'm working on the newsletter and added an edit to the opening sentences above :)

**Comment by stefaniebutland** [comment]
@jules32 Can you please help resolve conflicts and publish?

I tried but am stumped; burning it down & recloning did not resolve. I think because it's the `_site/`
 files. Weak mental model on my part. Not sure whether I need to accept head or base branch of each and how to choose them on cmd-line in the screenshot.

Screenshot 2024-01-22 at 11 30 34 AM

**Comment by stefaniebutland** [comment]
Julie's advice
> I would delete the _site files until you can successfully resolve conflict (don’t try to go line-by-line through .html files, just delete them. Then, after you’ve successfully merged without conflict, rebuild the site and push any new files.

Was still stumped. Going to reclone, create new branch for this post, add the `index.qmd` and logos image, render, push.

note to self: figure out and practice the commands in RStudio to update my older branch with what has been pushed to `Main`.  Just clicking `pull` in git tab does not accomplish this

**Comment by stefaniebutland** [comment]
merged in new branch, new PR to work around merge conflicts https://github.com/Openscapes/website-new/pull/87

**Pull request closed without merging**